### ISSUE TITLE
tflite_convert makes no difference to converted model after changing detection_postprocess.cc

### ISSUE THREAD

**Issue by sarmadidrees** [tensorflow/tensorflow#25311]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): **Linux Ubuntu 16.04**
- TensorFlow installed from (source or binary): **source**
- TensorFlow version (or github SHA if from source): **r1.12**

**Describe the issue**
I am converting SSD object detection model trained through object detection api. The only difference in my custom model and other pre-trained model is that the **detection_boxes** have 6 output coords instead of 4 box coords, I am also predicting the center coordinates of the bounding box to make the model more robust. It is performing great in frozen_graph.pb but as I convert it into .tflite format by using tflite_convert, it only gives 4 box coords whereas I need 6 box coords. But, eventually I figured out that I need to change the [detection_postprocess.cc](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/detection_postprocess.cc), but changes made in this file are not effecting the model, it seems like tflite_convert is ignoring this file while converting the model.

**Steps to reproduce the issue**
- tensorflow build from source and installed in a virtualenv
- make some changes in the [detection_postprocess.cc](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/detection_postprocess.cc)
- uninstall tensorflow from virtualenv
- again build tensorflow using bazel and install it using pip in same virtualenv
- run following command to convert model:
`tflite_convert \
  --output_file=detect.tflite \
  --graph_def_file=tflite_graph.pb \
  --input_shapes=1,300,300,3 \
  --input_arrays=normalized_input_image_tensor \
  --output_arrays='TFLite_Detection_PostProcess','TFLite_Detection_PostProcess:1','TFLite_Detection_PostProcess:2','TFLite_Detection_PostProcess:3' \
  --mean_values=128 \
  --std_dev_values=128 \
  --change_concat_input_ranges=false \
  --allow_custom_ops` 

**More information**
- tflite_graph.pb is obtained by using [export_tflite_ssd_graph.py](https://github.com/tensorflow/models/blob/master/research/object_detection/export_tflite_ssd_graph.py)

How to make sure that changes made in detection_postprocess.cc will effect the converted model using tflite_convert?

**Comment by sarmadidrees** [comment]
As, suggested by @achowdhery in this [comment](https://github.com/tensorflow/tensorflow/issues/21230#issuecomment-456803457), I already build the whole tensorflow from source after making changes to detection_postprocess.cc but still no success.
